### PR TITLE
CLC-4830 - CFV - Improve the accessibility on the search details page

### PR DESCRIPTION
**Pull request by christianv**
https://jira.ets.berkeley.edu/jira/browse/CLC-4830

* Add table scopes
* Show `search` instead of `go` or `return` when you're in the search text box on iOS. Thanks @nicolaasmatthijs
* Add labels

### CODE REVIEW

File: src/assets/templates/cars_activity.html [hunk 1]
```diff
@@ -7,7 +7,7 @@ <h2 class="cc-left">Activity</h2>
   </div>
 
   <div data-cc-spinner-directive>
-    <form name="cc_page_myfinances_account_filter" class="cc-page-myfinances-account-filter">
+    <form name="cc_page_myfinances_account_filter" class="cc-page-myfinances-account-filter" action="/" role="search">
```
Comment (nicolaasmatthijs) [inline src/assets/templates/cars_activity.html]: @christianv : If I remember correctly, the form should have an `id` attribute that contains the string `search` somewhere for the `Search` button to be shown on all iOS versions

Comment (nicolaasmatthijs) [inline src/assets/templates/cars_activity.html]: Apparently, you have to have "search" in the name or id attribute. I thought it was required on the form, but maybe it works on the input field as well.

Comment (christianv) [inline src/assets/templates/cars_activity.html]: @nicolaasmatthijs I used http://stackoverflow.com/questions/4864167/show-search-button-in-iphone-ipad-safari-keyboard/26253512#26253512 and tried it on the iOS simulator were everything worked correctly. (see screenshot in jira - https://jira.ets.berkeley.edu/jira/browse/CLC-4830

File: src/assets/templates/cars_activity.html [hunk 2]
```diff
@@ -54,7 +63,7 @@ <h2 class="cc-left">Activity</h2>
               <label class="cc-hidden cc-page-myfinances-print-label cc-print-show cc-left">
                 <strong>Search: </strong>
               </label>
-              <input type="text" placeholder="Search..." data-ng-model="search.$">
+              <input type="text" placeholder="Search..." data-ng-model="search.$" aria-label="Search" name="search">
```
Comment (nicolaasmatthijs) [inline src/assets/templates/cars_activity.html]: It looks like entering text here will immediately narrow down the list. Currently, there isn't anything announcing that that's happening though. Ideally, we would add an ARIA “polite” alert that announces this.

**Comment by nicolaasmatthijs** [comment]
Looks good. I left one more comment that we can deal with as part of a follow-up ticket.

File: src/assets/templates/cars_activity.html (hunk 2 above)
Comment (christianv) [inline src/assets/templates/cars_activity.html]: @nicolaasmatthijs totally agree - follow up at #3353

**Pull request merged**